lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -837,19 +821,18 @@ export class UserAgentApplication {
    */
   private openWindow(urlNavigate: string, title: string, interval: number, instance: this, resolve?: Function, reject?: Function): Window {
     // Generate a popup window
-    var popupWindow = this.openPopup(urlNavigate, title, Constants.popUpWidth, Constants.popUpHeight);
-
-    // if popupWindow is not valid, throw error
-    if (popupWindow == null) {
+    var popupWindow: Window;
+    try {
+      popupWindow = this.openPopup(urlNavigate, title, Constants.popUpWidth, Constants.popUpHeight);
```

Comment:
does this now throw if its null?

Reply:
Yes, before we were checking the nullity in this function. I have moved that to the previous function where we create the popupWindow and that function now throws if we do not create a window.